molecule/default/converge.yml

```diff
@@ -5,3 +5,4 @@
     - role: ansible-cartridge
   become: true
   become_user: root
+  gather_facts: no
```

Comment:
Let's use `false` value

Reply:
You can disable gather_facts by default,  move OS checks to separate block and call setup: explicitly

Reply:
Now OS checks are performed only if `ansible_os_family` fact is defined. 
I think that if one uses default scenario, he shouldn't change it just to check OS family.
But if facts were gathered we can additionally check OS family.